framework/Source/Sparkles.swift

```diff
@@ -197,4 +212,31 @@ extension Sparkles {
             --> output
         }
     }
+
+    private func reconnectFaceMaskInput(_ maskInput: PictureInput) {
+        let semaphore = DispatchSemaphore(value: 0)
+
+        maskInput.addTarget(lightExtractorEffect, atTargetIndex: 2)
+        Task {
+            await maskInput.processImage()
+            semaphore.signal()
+        }
+        semaphore.wait()
+    }
+}
+
+private extension UIImage {
+    static func makeEmptyMaskImage() -> UIImage {
+        let size = CGSize(width: 1, height: 1)
```

Comment:
Hi @yyjim here's the result when I pass nil as the maskImage:
<table>
<tr>
<th>With maskImage</th>
<th>maskImage = nil</th>
</tr>
<tr>
<td>

https://github.com/cardinalblue/GPUImage3/assets/40178645/aaef08a9-4a1e-4fa2-ae85-15cd63e13fc2

</td>
<td>

https://github.com/cardinalblue/GPUImage3/assets/40178645/794e8ecf-8d7d-40e3-ac89-44dcc9268e26

</td>
</tr>
</table>